conf-registration-web.iml

```diff
@@ -8,6 +8,16 @@
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
     <orderEntry type="library" scope="PROVIDED" name="sass-stdlib" level="application" />
+    <orderEntry type="module-library">
```

Comment:
@OzzieOrca I think you are correct Scotty.  This is an IntelliJ file and don't think we need it.  I use IntelliJ for all my development including any ERT.  Even if this file exists currently in master branch no changes should be made to it for this change (i.e. Event 433).